core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestTransformCorrelatedGlobalAggregationWithProjection.java

```diff
@@ -287,4 +289,39 @@ public void testWithPreexistingMask()
                                                         .right(project(ImmutableMap.of("non_null", expression(TRUE)),
                                                                 values(ImmutableMap.of("a", 0, "mask", 1)))))))));
     }
+
+    @Test
+    public void rewritesOnSubqueryWithBoolOr()
```

Comment:
nit: `testRewrites...`

Reply:
This will make naming convention inconsistent. I will make `refactor` PR outside this one.